pay-api/src/pay_api/models/invoice.py

```diff
@@ -37,6 +38,18 @@
 from .receipt import ReceiptSchema
 
 
+def determine_overdue_date(context):
+    """Determine the overdue date with the correct time offset."""
+    created_on = context.get_current_parameters()['created_on']
+    target_date = created_on.date() + relativedelta(months=2,
+                                                    day=1)
+    target_datetime = datetime.combine(target_date, datetime.min.time())
+    # Correct for daylight savings.
+    hours = target_datetime.astimezone(pytz.timezone('America/Vancouver')).utcoffset().total_seconds() / 60 / 60
+    target_date = target_datetime.replace(tzinfo=timezone.utc) + relativedelta(hours=-hours)
+    return target_date.replace(tzinfo=None)
```

Comment:
If i didn't do this, I would see weird behavour, would sometimes remove the hours.. but in the model unit test the model persists. tzinfo=None seems to make it always show the hours